packages/kit/types/app.d.ts

```diff
@@ -1,9 +1,15 @@
 import { ReadOnlyFormData, RequestHeaders } from './helper';
 import { ServerResponse } from './hooks';
+import { PrerenderOptions, SSRManifest } from './internal';
 
-export interface App {
-	init(): void;
-	render(incoming: IncomingRequest): Promise<ServerResponse>;
+export class App {
+	constructor(manifest: SSRManifest);
+	render(
+		incoming: IncomingRequest,
+		options?: {
```

Comment:
the `options` are only ever used internally, so we had moved them to `internal.d.ts` in order to simplify the method signature for adapter authors